src/test/java/uk/gov/companieshouse/api/accounts/controller/DebtorsControllerTest.java

```diff
@@ -27,6 +23,13 @@
 import uk.gov.companieshouse.api.accounts.utility.ApiResponseMapper;
 import uk.gov.companieshouse.api.accounts.utility.ErrorMapper;
 
+import javax.servlet.http.HttpServletRequest;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Can we please add explicit imports?

Reply:
Yeah, my intellij seems to be doing it for static imports, will fix.